utils.h

```diff
@@ -126,7 +126,8 @@
 #define VERSYM_HIDDEN 0x8000
 #define VERSYM_VERSION 0x7fff
 
-static const Elf_Versym NO_VERSION_INFO = 0xffff;
+static constexpr uintptr_t LINUX_PAGE_SIZE = 0x1000;
```

Comment:
Thanks for removing the magic number! FYI, `sysconf(_SC_PAGE_SIZE)` is the proper way to obtain this, but hard-coding the number would be enough.